static/css/taxbrain.css

```diff
@@ -8440,6 +8440,18 @@ label {
   border-right: 2px solid #dddddd;
   padding-left: 30px;
 }
+.table-check-all {
```

Comment:
I had intended to use them in the row at the top of the table in depreciation of btax (the row that lets you check all).  I'll make sure it is still referenced.